cromwell-backend/src/main/scala/cromwell/backend/config/BackendConfiguration.scala

```diff
@@ -0,0 +1,47 @@
+package cromwell.backend.config
+
+import com.typesafe.config.ConfigFactory
+
+import scala.collection.JavaConversions._
+
+/**
+  * Defines a backend configuration.
+  * @param name Backend name.
+  * @param initClass Initialization class for the specific backend.
+  * @param `type` Backend type.
+  */
+case class BackendConfigurationEntry(name: String, initClass: String, `type`: String)
+
+object BackendConfiguration {
+  val config = ConfigFactory.load()
+  val backendCfg = config.getConfig("backend")
+  val defaultBackend = backendCfg.getString("default")
+  val backendProviders = backendCfg.getConfigList("providers")
+  val backendList = backendProviders.map(entry =>
```

Comment:
It explicitly says to prefer the infix style for higher order functions. "As noted, methods which take functions as parameters (such as map or foreach) should be invoked using infix notation"

And then for the `.map(...` style it says "This style is not the accepted standard!"